master/manifests/site.pp

```diff
@@ -284,6 +284,10 @@
 }
 
 jenkins::plugin {
+  'workflow-step-api': ;
```

Comment:
At the time I added that line, the Puppet module that installed Jenkins did not install plugin dependencies automatically; they all had to be specified explicitly (sorry, I don't have a documentation reference to hand). This is a dependency of the SSH Agent Plugin per https://wiki.jenkins-ci.org/display/JENKINS/SSH+Agent+Plugin